apps/toolkit-app/app/auth/mutations/signup.ts

```diff
@@ -0,0 +1,19 @@
+import { prisma } from "../../../db/index"
+import { SecurePassword } from "@blitzjs/auth"
+
+export default async function signup(input, ctx) {
+  const blitzContext = ctx
+
+  const hashedPassword = await SecurePassword.hash((input.password as string) || "test-password")
+  const email = (input.email as string) || "test" + Math.random() + "@test.com"
+  const user = await prisma.user.create({
+    data: { email, hashedPassword, role: "user" },
+    select: { id: true, name: true, email: true, role: true },
+  })
+
+  await blitzContext.session.$create({
+    userId: user.id,
```

Comment:
We also have to pass `role` to $create (but I think there's something broken with types now)